mountpoint-s3-fs/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "mountpoint-s3-fs"
 # See `/doc/PUBLISHING_CRATES.md` to read how to publish new versions.
-version = "0.7.1"
+version = "0.7.2"
```

Comment:
`Cargo.lock` should also be included in the commit whenever there are changes to the `Cargo.toml` files.